src/mixer.c

```diff
@@ -860,7 +860,12 @@ Mix_Chunk *Mix_LoadWAV_RW(SDL_RWops *src, int freesrc)
             return(NULL);
         }
 
-        chunk->abuf = SDL_realloc(wavecvt.buf, wavecvt.len_cvt);
+        Uint8 *const resized_buf = SDL_realloc(wavecvt.buf, wavecvt.len_cvt);
```

Comment:
This won't compile on many systems, the declaration needs to be moved to the top of the block
`const Uint8 *resized_buf;`

Reply:
Fixed.